src/integration/__tests__/__fixtures__/knownV2Failures.json

```diff
@@ -948,5 +948,12 @@
   "ebb240b91abb54dde55e46da86a0a885": null,
   "acf634b76c571c63c354f9203e7757f1": null,
   "e435b0066fae9cdb0cddd2146c1e383f": null,
-  "810fe4219c35f8c1198fcbd6bdbed29e": null
+  "810fe4219c35f8c1198fcbd6bdbed29e": null,
+  "042c99463b706a2a8a35d4b0a5337c71": null,
+  "7114b24c69da6fdf8a952cc998dded17": null,
+  "5feb6d845487b8b8d696e0cce082eb2b": null,
+  "491575e8d9b9f682de1cb678fd351c4b": null,
+  "e089beca2667c383e7e2c99c42e8750f": null,
+  "bd37ef982226143b0d68d972b5fee276": null,
+  "5d6a47112815cab926fb8f2036b94039": null
```

Comment:
Some persisted My Bids queries became invalid because we migrated the `floorSellingPirce` and `onlineAskingPrice` fields to MP's `Money` type. The feature hasn't been released yet so _I think_ it's safe to ignore here. @zephraph would like to see if you think it's ok.

They all failed for the same reason:

![Screen Shot 2020-11-30 at 10 29 10 PM](https://user-images.githubusercontent.com/796573/100696690-237ae400-3362-11eb-8a82-26ae5ea0b2fc.png)